tests/validation/cattlevalidationtest/core/test_container_run_option.py

```diff
@@ -189,12 +187,7 @@ def test_container_run_with_options_1(client, test_name, managed_network,
                     "result": False})
 
     configs.append({"docker_param_name": "Config.Cmd",
-                    "docker_param_value": command,
-                    "docker_value": None,
-                    "result": False})
-
-    configs.append({"docker_param_name": "Args",
-                   "docker_param_value": command_args,
+                    "docker_param_value": command[0] + " " + command[1],
```

Comment:
This is checking against the docker inspect? I would have expected the inspect to have an array for Cmd as well